src/SprykerSdk/Zed/AppSdk/Business/AsyncApi/Builder/AsyncApiBuilder.php

```diff
@@ -390,4 +389,28 @@ protected function orderAsyncApiElements(array $asyncApi): array
 
         return $asyncApi;
     }
+
+    /**
+     * @param string $targetFile
+     * @param array $asyncApi
+     *
+     * @return void
+     */
+    private function updateFile(string $targetFile, array $asyncApi): void
```

Comment:
```suggestion
    protected function updateAsyncApi(string $targetFile, array $asyncApi): void
```
We try to avoid using private, please only use protected.